certified-connectors/Converter/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Api-Key (free on \"https://power2apps.de/apikey\")",
+          "description": "\"Api-Key (free on \"https://power2apps.de/apikey\")\" f\u00fcr diese API",
+          "tooltip": "Stellen Sie \"Api-Key (free on \"https://power2apps.de/apikey\")\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
```

Comment:
@stsiek This is the default iconBrandColor for all the independent publisher connectors, Please make sure it is the background color of your connector icon.